x/bank/keeper/send.go

```diff
@@ -305,17 +317,30 @@ func (k BaseSendKeeper) setBalance(ctx sdk.Context, addr sdk.AccAddress, balance
 // any of the coins are not configured for sending.  Returns nil if sending is enabled
 // for all provided coin
 func (k BaseSendKeeper) IsSendEnabledCoins(ctx sdk.Context, coins ...sdk.Coin) error {
+	if len(coins) == 0 {
+		return nil
+	}
+	store := ctx.KVStore(k.storeKey)
+	haveDefault := false
+	var defaultVal bool
```

Comment:
This is an incorrect implementation. Again, we should not be referencing or using the legacy params at all.

Reply:
Here again, the `Params.DefaultSendEnabled` value is not depreciated and is still to be used. It is the fallback for cases when there is no `SendEnabled` entry for a denom. That functionality hasn't changed. The extra code in here just makes it so that the bank module's `Params` only need to be looked up once during this function call (rather than once per coin).